### PR TITLE
updating homework

### PR DESCRIPTION
**Pull request by jjame369**
updated homework

### CODE REVIEW

File: students/Justin_Jameson/lesson01/assignment/inventory_management/furniture_class.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+                         description,
+                         market_price,
+                         rental_price)
+
+        self.material = material
+        self.size = size
+        self.output_dict = {}
+
+    def return_as_dictionary(self):
+        """"can I get rid of the replicated ones from the original class?"""
```
Comment (majstorpoki) [inline students/Justin_Jameson/lesson01/assignment/inventory_management/furniture_class.py:34]: Can you?

File: students/Justin_Jameson/lesson01/assignment/inventory_management/main.py [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+import sys
+import market_prices
+import inventory_class
+import furniture_class
+import electric_appliances_class
+
+
+
+def main_menu(user_prompt=None):
+    """"This method is the menu for the program. The intent is to
```
Comment (majstorpoki) [inline students/Justin_Jameson/lesson01/assignment/inventory_management/main.py:22]: I really like your comments

File: students/Justin_Jameson/lesson01/assignment/test_unit.py [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+    """Tests for main.py:
+    1. does each prompt work in 'main_menu'.
+    2. when adding content to 'add_new_item', item is added to the dictionary.
+    3. 'item_info' either prints the dictionary or 'Item not found in Inventory' """
+
+    def test_main(self):
+
+
+class InventoryTest(unittest.TestCase):
+    """Tests for inventory_class
```
Comment (majstorpoki) [inline students/Justin_Jameson/lesson01/assignment/test_unit.py:20]: I like this

File: students/Justin_Jameson/lesson01/assignment/test_unit.py [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+        description = 'new'
+        market_price = 24
+        rental_price = 70
+        self.assertEqual(test_dic, output_dict)
+        
+
+class FurnitureTest(unittest.TestCase):
+    """Tests for furniture_class
+    1. can take 6 input items and create a dictionary"""
+    pass
```
Comment (majstorpoki) [inline students/Justin_Jameson/lesson01/assignment/test_unit.py:37]: What is purpose of this?

File: students/Justin_Jameson/lesson02/assignment/src/charges_calc.py [hunk 5]
```diff
@@ -0,0 +1,88 @@
+# -------------------------------------------------#
+# # Title: charges calculator for Inventory management.
+# # Dev:   unknown
+# # Date:  4/17/2019
+# # ChangeLog: (Who, What)
+# Justin Jameson
+# correct ouput to output line 20 'ouptu JSON file'
```
Comment (majstorpoki) [inline students/Justin_Jameson/lesson02/assignment/src/charges_calc.py:7]: Thanks.

File: students/Justin_Jameson/lesson02/assignment/src/charges_calc.py [hunk 6]
```diff
@@ -0,0 +1,88 @@
...
+            logging.debug('cycling through calculate_add.. try block')
+            rental_start = datetime.datetime.strptime(value['rental_start'], '%m/%d/%y')
+            logging.debug('made it through rental_start')
+            rental_end = datetime.datetime.strptime(value['rental_end'], '%m/%d/%y')
+            logging.debug('made it through rental_end')
+            value['total_days'] = (rental_end - rental_start).days
+            logging.debug('made it through value total_days')
+            value['total_price'] = value['total_days'] * value['price_per_day']
+            logging.debug('made it through total_price')
+            # value['sqrt_total_price'] = math.sqrt(value['total_price'])
```
Comment (majstorpoki) [inline students/Justin_Jameson/lesson02/assignment/src/charges_calc.py:64]: Think how you could log particular errors that could happen (e.g., missing field, end date smaller than beginning date...) and what would be severity of these log messages.

**Review by majstorpoki — APPROVED** [review]

**Review by ObjectiveJoe — APPROVED** [review]

**Pull request merged**